actions-generate-docs.py

```diff
@@ -113,6 +214,11 @@ def main():
     doc_script = os.path.join(this_folder, "scripts", "build_docs.sh")
     output_path = os.path.join(root_path, "_build")
     source_path = os.path.join(root_path, "docs")
+    config_paths = [
+        os.path.join(this_folder, "jekyll", "_config.yml"),
+        os.path.join(root_path, "jekyll", "_config.yml"),
+        os.path.join(root_path, "jekyll_config.yml"),
+    ]
 
     # first figure out if we are in a PR.
     if os.environ.get("GITHUB_EVENT_NAME") == "pull_request":
```

Comment:
Do we know for a fact that we'll never need to run your `cleanup_image_il8n` other than pull request? Frankly, I know enough, but since we're invoking the `build_docs.sh` script in 2 different places, I would naively think the cleanup code might also be called in 2 different places.

Reply:
Great catch - I've added a call to `cleanup_image_i18n` in the `main` branch logic branch as well.